### DISCUSSION TITLE
SP-API Guard

### DISCUSSION THREAD

**Discussion by mtessar** [amzn/selling-partner-api-models#3538] [General]
Are all public SP-API application developers required to submit results from the SP API Guard, or just those using PII?

**Comment by sungolivia** [comment]
when you mention submit, you mean submitting the report findings?
![Screenshot 2023-03-06 at 5 19 56 PM](https://user-images.githubusercontent.com/7606170/223294420-dba6fabf-d108-4ee8-b76c-dc4a179d7bf0.png)

**Reply by mtessar** [reply]
Hi, what I am trying to determine is whether or not all public API developers are required to submit results to Amazon. It seems like this is more for application developers who have PII/use the restricted data tokens but I am not sure.
